docs/sphinx/cookbook/shared_memory_allocators.rst

```diff
@@ -74,14 +74,61 @@ uses the MPI3 API for its Shared Memory mechanisms that allow processes to commu
 To use Umpire's MPI3 Shared Memory allocators, the ``UMPIRE_ENABLE_MPI3_SHARED_MEMORY`` flag 
 should be set to ``On``. Note that if you are using MPI3 Shared Memory, then MPI must be enabled.
 
-See the bottom of this page for a full example of how to use Shared Memory Allocators with Umpire.
+To create an allocator with the MPI3 Shared Memeory resource, you can do the following:
 
-Full IPC Shared Memory Recipe
------------------------------
+.. code-block:: cpp
+
+   auto traits{umpire::get_default_resource_traits("SHARED::MPI3")};
+   auto node_allocator{rm.makeResource("SHARED::mpi3_alloc", traits)};
+
+See the bottom of this page for a full example of how to use MPI3 Shared Memory Allocators with Umpire.
+
+Using Both IPC and MPI3 Shared Memory Allocators
+------------------------------------------------
+
+It is possible to enable both IPC and MPI3 Shared Memory Allocators at the same time.
+
+To create these Shared Memeory allocators, you can do the following:
+
+.. code-block:: cpp
+
+   auto mpi3_traits{umpire::get_default_resource_traits("SHARED::MPI3")};
+   // or
+   auto ipc_traits{umpire::get_default_resource_traits("SHARED::POSIX")};
+
+   // then create an allocator:
+   auto mpi3_node_allocator{rm.makeResource("SHARED::mpi3_alloc", traits)};
```

Comment:
If IPC is the default, would this work? Or do you need "SHARED::MPI3::<some allocator name>"?